examples/tts/radtts.py

```diff
@@ -21,6 +26,12 @@
 from nemo.utils.exp_manager import exp_manager
 
 
+def handle_pdb(sig, frame):
+    import pdb
+
+    pdb.Pdb().set_trace(frame)
```

Comment:
it would be better to convey clear thread error information to users instead of calling pdb here.

Reply:
This is to debug hung process - there is no error information here.

Reply:
Removed just in case.